lib/component-repository/src/routes/virtual-components/CreateComponentConfig.js

```diff
@@ -1,3 +1,36 @@
+const ComponentConfig = require('../../models/ComponentConfig');
+const mongoose = require('mongoose');
+
 module.exports = async function (req, res, next) {
-    return next();
+  const { authClientId } = req.body;
+  const { componentVersionId } = req.params;
+  const tenant = req.user.isAdmin ? req.body.tenant : req.user.tenant;
+
+  if (!mongoose.Types.ObjectId.isValid(componentVersionId)) {
+    return res
+      .status(400)
+      .send({ errors: [{ message: 'Invalid id', code: 400 }] });
+  }
+  const currentVersionCompConfig = await ComponentConfig.findOne({
+    componentVersionId: componentVersionId,
+    tenant: tenant,
```

Comment:
since the variables and parameters have the same name you can just use the shorthand anotation:
componentVersionId,
tenant